frontend/src/components/GalleryItems.tsx

```diff
@@ -14,11 +14,9 @@ export default function GalleryItems(props: GalleryItemsProps) {
                 {props.country.translations.deu.common} <br/>
                 {"Capital : " + props.country.capital} <br/>
                 {/*{"Language " + props.country.languages}<br/>*/}
-                {"Population " + props.country.population}<br/><br/>
+                {"Population: " + props.country.population}<br/><br/>
```

Comment:
instead of yousing `<br/>` u would suggest using css + html 
```JSX
            <div>  {props.country.translations.deu.common} </div>
               <div> {"Capital : " + props.country.capital} </div>
                {/*{"Language " + props.country.languages}<br/>*/}
               <div> {"Population " + props.country.population}</div>
                <div> {"Population: " + props.country.population}</div>
```
have'nt tried it, possibly u have to use a littlebit css after this.